subgraphs/nft-market/mappings/index.ts

```diff
@@ -13,10 +13,11 @@ import {
 } from "../generated/ERC721NFTMarketV1/ERC721NFTMarketV1";
 import { toBigDecimal } from "./utils";
 import { updateCollectionDayData, updateMarketPlaceDayData } from "./utils/dayUpdates";
-import { fetchName, fetchSymbol, fetchTokenURI } from "./utils/erc721";
+import { fetchBunnyId, fetchName, fetchSymbol, fetchTokenURI } from "./utils/erc721";
 
 // Constants
 let ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+let PANCAKE_BUNNIES_ADDRESS = "0x60935f36e4631f73f0f407e68642144e07ac7f5e";
```

Comment:
We should start thinking about doing a global configuration file to support testnet and mainnet based on `dataSource.network()` available at the runtime